config.yml.default

```diff
@@ -112,6 +112,8 @@ challenge:                   # Incoming challenges.
   min_increment: 0           # Minimum amount of increment to accept a challenge.
   max_base: 315360000        # Maximum amount of base time to accept a challenge. The max is 315360000 (10 years).
   min_base: 0                # Minimum amount of base time to accept a challenge.
+  max_days:                  # Maximum number of days per move to accept a challenge for a correspondence game.
+  min_days: 1                # Minimum number of days per move to accept a challenge for a correspondence game.
```

Comment:
I wasn't quite sure about the best values to put here. 1 is a reasonable default for the minimum, but it seemed wrong to put some arbitrary number to represent "unlimited" or "no maximum". Leaving it empty is interpreted as no upper bound (allowing unlimited games), just as not putting a `max_days` field at all.

I could have also commented them out.